demos/git-client/README-ssh-host-key-configuration.md

```diff
@@ -0,0 +1,17 @@
+# Configure Git client plugin
+
+## Sample ssh host key verification policy configuration
```

Comment:
this should be in the README.md in this directory. You can access different yaml snippets in tests by referring to their index:
https://github.com/jenkinsci/configuration-as-code-plugin/blob/097dcd0951febe91f3d4e468e9394bfdde3d0f87/integrations/src/test/java/io/jenkins/plugins/casc/core/HudsonPrivateSecurityRealmConfiguratorTest.java#L31

Reply:
Thanks for the pointer!  I've added a test of the host key verification configuration and have moved the host key setting into the git-client README.